kolibri/core/assets/src/views/ExamReport/PageStatus.vue

```diff
@@ -10,6 +10,17 @@
         <h1 class="title">
           <KLabeledIcon icon="person" :label="userName" />
         </h1>
+        <div v-if="windowIsSmall" class="completion-status">
```

Comment:
This component is deleted in #8807 - so I may want some help from you to resolve the inevitable merge conflict and ensure these mobile responsiveness changes are preserved.